_profiles/drupal/resource-versioning/index.md

```diff
@@ -0,0 +1,273 @@
+| `d`      | `g`              | `h`            | `a`                   | `e`                 | `b`, `c`             | `e`                       |
+| `e`      | `g`              | `h`            | `a`                   | `g`                 | `d`                  | `f`                       |
+| `f`      | `g`              | `h`            | `e`                   | `g`                 | `e`                  | `g`                       |
+| `g`      | no link          | `h`            | `e`                   | no link             | `f`                  | `h`                       |
+| `h`      | `g`              | no link        | `g`                   | no link             | `g`                  | no link                   |
+
+> Note: In this example, `f` has both a `predecessor-version` and
+> `prior-working-copy` link to `e` because `e` was a version and `e` was also the
+> revision to which changes could be applied prior to `f`'s creation.
+
+# Version History
+
+A server **MAY** provide a "version history" endpoint. The primary data
+of a response document from a version history endpoint must be a collection of
+resource objects.
+
+Unless an `id` contains version information, the `type` and `id` members of each
+resource object in the collection **MUST** be the same.
+
+If provided, resource objects' `self` links **MUST NOT** be the same.
+
+# Version Negotiators
+
+## ID-Based Version Negotiator
+
+This profile establishes the `id` version negotiator. An `id`-based version
+identifier is composed of two segments—the `id` version negotiator and a single
+version argument. Any colons (U+003A COLON, “:”) present in the version
+identifier after the first occurrence **MUST** be interpreted as part of the
+single version argument and **MUST NOT** be interpreted as a segment delimiter.
+
+The resource version returned for any given version argument in an `id`-based
+version identifier **MUST NOT** change over time.
+
+> Note: This profile is agnostic about the format of the version argument in
+> `id`-based version identifiers. For example, one server may use integers as
+> revision IDs, another may use UUIDs and yet another may use content-based
+> hashes.
+
+## Relative Version Negotiator
+
+This profile establishes the `rel` version negotiator. A `rel`-based version
+identifier is composed of two or more segments. The first segment **MUST** be
+`rel` and the following version arguments describe a resource version that is
+relative to the version history.
+
+The resource version returned for any given version argument in a `rel`-based
+version identifier **MAY** change over time.
+
+The `rel` version negotiator has the following valid version argument strings:
+
+  - `latest-version`: requests the latest default revision of a resource.
+  - `working-copy`: requests revision of a resource to which changes can be
+    made.
+
+If any of the following version arguments is received in a `rel`-based version
```

Comment:
[Interoperable implementations requirement] It's not clear to me if servers are allowed to define their own arguments to version negotiators defined in this profile. E.g., could a server support `rel:xxx`? (Note: if a server can, that also limits your ability to extend these argument lists, except to define the four "reserved" first argument names that you give below.)

Reply:
No, a server shouldn't support any custom rel-based negotiators. I've clarified that in: 6bc95be

Good catch!